src/DropdownMenu/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import DropdownMenu from './DropdownMenu';
```

Comment:
minor: If I'm not mistaken, when using only `export` you can do `export X from './X` (it doesn't work only for `export default`)